enterprise/signals.py

```diff
@@ -106,6 +107,58 @@ def default_content_filter(sender, instance, **kwargs):     # pylint: disable=un
         instance.save()
 
 
+@receiver(post_save, sender=EnterpriseCustomerUser)
+def assign_or_delete_enterprise_admin_role(sender, instance, **kwargs):     # pylint: disable=unused-argument
+    """
+    Assign or delete enterprise_admin role for EnterpriseCustomerUser when updated.
+
+    This only occurs if a PendingEnterpriseCustomerAdminUser record exists.
+    """
+    if instance.user:
+        enterprise_admin_role, __ = SystemWideEnterpriseRole.objects.get_or_create(name=ENTERPRISE_ADMIN_ROLE)
```

Comment:
Wouldn't this create an admin role for any `EnterpriseCustomerUser` that gets saved?

Reply:
Never mind, took me a second to realize that we weren't assigning it to the user yet